docs/learn/storage/aws-ami.md

```diff
@@ -0,0 +1,121 @@
+
+![AMI search bar](/img/learn/ami_search.png)
+
+The page will bring you to the search results. Click "Community AMIs" tab in the search result page and select the AMI you want to use in the list.
+
+![AMI search result](/img/learn/ami_select.png)
+
+### Allow inbound connections
+
+When launching a new EC2 instance in the AWS console, you can either create a new security group for the instance or choose existing security group. Either way, you must add inbound rules to allow connections to the ports that Kaia nodes use to communicate each other.
+
+Navigate to the EC2 instance page in the AWS console, find associated security groups in the "Security" tab. You should add inbound rules for port 32323-32324.
+
+| IP version | Type        | Protocol | Port range     | Source   |
+|------------|-------------|----------|----------------|----------|
+| IPv4       | Custom TCP  | TCP      | 32323 - 32324  | 0.0.0.0/0|
+| IPv4       | Custom UDP  | UDP      | 32323          | 0.0.0.0/0|
+
+## Post-Launch Instance Preparation and Setup
+
+### Warmup Amazon EBS volumes
+
+The Amazon EBS volumes that were created from snapshots (AMI would be one case), the storage blocks must be pulled down from Amazon S3 and written to the volume before they can be accessed. This causes a significant overhead for disk operations the first time each block is accessed. Volume performance is recovered after all blocks have been downloaded and written to the volume. For more details, see [Initialize Amazon EBS volumes](https://docs.aws.amazon.com/ebs/latest/userguide/ebs-initialize.html).
+
+In order to get our volume ready, we can run a task to read all blocks.
+
+```bash
+$ sudo yum install -y fio
+$ sudo fio --filename=/dev/nvme1n1 --rw=read --bs=128k --iodepth=32 --ioengine=libaio --direct=1 --name=volume-initialize
+```
+
+:::note
+
+This task to warmup Amazon EBS volume would take a long time depending on the data size. Refer to `fio` output for the ETA.
+
+:::
+
+### Start `kend` service
+
+In the EC2 instance, Kaia CLI client and the chaindata are installed. Also, `kend`, the script to start/terminate EN, is installed as a service. You can check the `kend` service's status by the following command.
+
+```bash
+$ sudo service kend status
+```
+
+If the service is not running, try to restart it.
+
+```bash
+$ sudo service kend restart
+```
+
+If the service restarted and the EN has started successfully, you can check its logs in the path `/var/kend/logs/kend.out`.
+
+```bash
+$ tail -f /var/kend/logs/kend.out
+```
+
+Kaia provides a CLI client `ken console`. You can interact with the Kaia node using the `ken console` via multiple endpoints, and one option is to use IPC (inter-process communication). The IPC file `kaia.ipc` is located in the data directory on an EN, in our case `/var/kend/data`. Thus in order to use `ken console`:
```

Comment:
The two ipc files 'kaia.ipc' and 'klay.ipc' coexist?

Reply:
No, my bad. But I'm afraid that there're some `kaia.ipc` in docs..
Anyway, updated. PTAL.

Reply:
You mean every instance of 'kaia.ipc' in docs needs to be changed to 'klay.ipc' all together?

Reply:
Yes, currently we use the name of the IPC file as `klay.ipc`.